packages/lumx-react/src/components/list/ListItem.tsx

```diff
@@ -144,9 +151,19 @@ const ListItem: React.FC<ListItemProps> = ({
             onClick={onItemSelected}
             onKeyDown={onKeyDown()}
         >
-            {before && <div className={`${CLASSNAME}__before`}>{before}</div>}
-            <div className={classNames(`${CLASSNAME}__content`)}>{children}</div>
-            {after && <div className={`${CLASSNAME}__after`}>{after}</div>}
+            {useAsLink ? (
+                <a {...linkProps}>
+                    <ListItemContent before={before} after={after} baseClassName={CLASSNAME}>
+                        {children}
+                    </ListItemContent>
```

Comment:
I don't think we need the `ListItemContent` component. I'd rather repeat the three lines of list item content.
Or we could do 
```jsx
const content = (
    <>
        {before && <div className={`${CLASSNAME}__before`}>{before}</div>}
        className={classNames(`${CLASSNAME}__content`)}>{children}</div>	                            
        {after && <div className={`${CLASSNAME}__after`}>{after}
   </>
);
```
And then:
```jsx
  {useAsLink ? <a...>{content}</a> : <div ...>{content}</div> }
```

Reply:
Yes, why not :)